sites/example-project/src/components/ui/Formatting/CurrencyFormatGrid.svelte

```diff
@@ -0,0 +1,109 @@
+<script>
+  import "./format-grid.css";
+  import { blur } from 'svelte/transition';
+  import { defaultExample, formatExample } from "$lib/modules/formats";
+  export let formats;
+  let selectedCurrency;
+</script>
+
+<select bind:value={selectedCurrency}>
+  <option name=usd id=usd value=usd>USD - United States Dollar</option>
+  <option name=eur id=eur value=eur>EUR - Euro</option>
+  <option name=jpy id=jpy value=jpy>JPY - Japanese Yen</option>
+  <option name=gbp id=gbp value=gbp>GBP - Pound Sterling</option>
+  <option name=aud id=aud value=aud>AUD - Australian Dollar</option>
+  <option name=cad id=cad value=cad>CAD - Canadian Dollar</option>
+  <option name=chf id=chf value=chf>CHF - Swiss Franc</option>
+  <option name=cny id=cny value=cny>CNY - Renminbi</option>
+  <option name=hkd id=hkd value=hkd>HKD - Hong Kong Dollar</option>
+  <option name=nzd id=nzd value=nzd>NZD - New Zealand Dollar</option>
+  <option name=sek id=sek value=sek>SEK - Swedish Krona</option>
+  <option name=krw id=krw value=krw>KRW - South Korean Won</option>
+  <option name=sgd id=sgd value=sgd>SGD - Singapore Dollar</option>
+  <option name=nok id=nok value=nok>NOK - Norwegian Krone</option>
+  <option name=mxn id=mxn value=mxn>MXN - Mexican Peso</option>
+  <option name=inr id=inr value=inr>INR - Indian Rupee</option>
```

Comment:
Seems like the currencies without a match in pre-built formats will show an empty table when selected.  Should we exclude these?

Got me thinking, if we were to add all of these all, we could end up having a lot of unintended formatting.  No data to back this, but my gut feeling is column names ending with _nok, _rub, _try, _cop, _sar etc are less likely to be actual containing the currency values and more likely to be a random column suffix.

Reply:
@ud3sh great point. 

I had left these ones blank until figuring out how we could add them all without creating a huge list of formats (it would be 35 currencies x 13 format variations = 455 formats).

I was trying to figure out if there's a way to dynamically create the 13 variations so we don't need to create a giant object containing all 455 (e.g., you click `MXN` in the drop-down and it swaps in the `mxn` prefix and currency symbol). The tricky part would be what happens when someone uses one of the variations in their code (e.g., `sales_mxn2b`).

On second thought though, your point on the column names is good. If people want to use a specific currency we don't have, they can also add a custom number format and use that.

I propose supporting the top 10 currencies in this round. Let me know if you have any ideas on creating the list of formats - otherwise I'll set up something quick to generate them just for this time.